homeassistant/components/meraki/device_tracker.py

```diff
@@ -73,7 +73,7 @@ async def post(self, request):
         if data["secret"] != self.secret:
             _LOGGER.error("Invalid Secret received from Meraki")
             return self.json_message("Invalid secret", HTTPStatus.UNPROCESSABLE_ENTITY)
-        if data["version"] != VERSION:
+        if data["version"] != VERSION and data["version"] != VERSION2:
```

Comment:
```suggestion
        if data["version"] not in (VERSION, VERSION2):
```

Reply:
Alternatively:

```python
from awesomeversion import AwesomeVersion
...
MIN_API_VERSION = "2.0"
MAX_API_VERSION = "2.1"
...
if MIN_API_VERSION <= AwesomeVersion(data["version"]) <= MAX_API_VERSION:
```

Reply:
Yes, if that's the check we want to do. Note that's it's not the same logic as the current PR code.